WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -342,9 +342,11 @@ def post_results
           body += "#{record_strs.join(", ")}.  \n" # Trailing spaces for markdown give us a <br>
         end
       end
-      comp.update!(results_posted_at: Time.now)
-      comp.competitor_users.each { |user| user.notify_of_results_posted(comp) }
-      comp.registrations.accepted.each { |registration| registration.user.notify_of_id_claim_possibility(comp) }
+      unless competition.results_posted?
```

Comment:
It should be `comp` here, that's why the [build fails](https://travis-ci.org/thewca/worldcubeassociation.org/builds/514684560#L4584) =)